web/forms.py

```
@@ -255,6 +256,51 @@ def save(self, request):
         return user
 
 
+class CampaignForm(forms.ModelForm):
+    class Meta:
+        model = Campaign
+        fields = ["title", "description", "funding_goal", "itemized_budget", "video_url", "image"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)